src/introduction/abtnode-access/index.md

```diff
@@ -26,4 +28,14 @@ Next, scan the generated QRCode using your ArcBlock ABT Wallet. It will authoriz
 
 ![scan-abt-node](./images/scan-abt-node.png)
 
-Once authorized, you should have access to the ABT Node dashboard.
+After authorization you will have ownership of the ABT Node. You will have full access to all components of the respective ABT Node. You can perform administration tasks not limiting to the following :
+- Monitor ABT Node KPIs (CPU, memory, storage)
+- use the Marketplace to install Blocklets
+- update / remove Blocklets
+- configure routing to the different apps
+- add / remove team member
+- troubleshoot using logs
+- provide SSH access to the ABT node
```

Comment:
Should we capitalize the bullet list items?

Reply:
fixed